### ISSUE TITLE
we need to be able to define a mix of input files

### ISSUE THREAD

**Issue by johannct** [COINtoolbox/RESSPECT#153]
Whether objects come from RESSPECT simu, SNPCC, ELASTICC, etc..., surveys is not relevant anymore once they are read into light_curve objects. We need a factory kind of function that creates such a list of light_curve objects from a file listing in 2 columns which survey and a path corresponding to an object of this survey (the simplest API I can think of right now).
This will be useful for unit testing, but I see how to trivially import it in the library, not how to turn it into something that the python scripts can call (should still be straightforward)
This will also allow to make sure that the DB interface is agnostic to the survey specific file datasets.

**Comment by emilleishida** [comment]
forget about the RESSPECT data set. Those were perfect sims that were used in the beginning only for first prototypes. 
I do not believe they will be useful in the future

**Comment by johannct** [comment]
is the related code planned for removal from the repository? Perfect sims handy may prove useful in future stages of code development, I would say....

**Comment by emilleishida** [comment]
yes, it is planned for removal .. it has not been tested or updated for a long time
I guess that after we refactor everything it should be easier to add it again. But for now I am not sure if the effort necessary to test it and keep it up to date is worth the effort. 

I already removed most of it, I can restore if that is the case, but I really do not think it is worth it
